src/applications/PrepareDataset.py

```diff
@@ -202,262 +231,217 @@ def copyFilesToCorrectLocation(interimOutputDir, finalSubjectOutputDir, subjectI
-                interimOutputDir_actual,
-                finalSubjectOutputDir_actual,
-                subject_id_timepoint,
-            )
-            # if there are any errors, then store the subjectid and timepoint
-            if runBratsPipeline:
-                subjects_with_bratspipeline_error = pd.concat(
-                    [
-                        subjects_with_bratspipeline_error,
-                        pd.DataFrame(
-                            {
-                                "SubjectID": subject_id,
-                                "Timepoint": timepoint,
-                            },
-                            index=[0],
-                        ),
-                    ]
-                )
-            else:
-                for modality in ["T1", "T1GD", "T2", "FLAIR"]:
-                    count = _read_image_with_min_check(outputs[modality])
-                    # if there are any negative values, then store the subjectid, timepoint, modality and count of negative values
-                    if count > 0:
-                        subjects_with_negatives = pd.concat(
-                            [
-                                subjects_with_negatives,
-                                pd.DataFrame(
-                                    {
-                                        "SubjectID": subject_id,
-                                        "Timepoint": timepoint,
-                                        "Modality": modality,
-                                        "Count": count,
-                                    },
-                                    index=[0],
-                                ),
-                            ]
-                        )
-                        negatives_detected = True
-        # store the outputs in a dictionary when there are no errors
-        if not negatives_detected:
-            output_df_for_csv = pd.concat(
-                [
-                    output_df_for_csv,
-                    pd.DataFrame(
-                        {
-                            "SubjectID": subject_id,
-                            "Timepoint": timepoint,
-                            "T1": outputs["T1"],
-                            "T1GD": outputs["T1GD"],
-                            "T2": outputs["T2"],
-                            "FLAIR": outputs["FLAIR"],
-                        },
-                        index=[0],
-                    ),
-                ]
-            )
+        # check if the files exist already, if so, skip
+        if not runBratsPipeline:
+            return
+
+        command = (
+            bratsPipeline_exe
+            + " -t1 "
+            + row[parsed_headers["T1"]]
+            + " -t1c "
+            + row[parsed_headers["T1GD"]]
+            + " -t2 "
+            + row[parsed_headers["T2"]]
+            + " -fl "
+            + row[parsed_headers["FLAIR"]]
+            + " -o "
```

Comment:
Just ensuring that _in case_ someone has an older version of the `BraTSPipeline`, we want skull stripping to be specifically **not** invoked.
```suggestion
            + " -s 0 -o "
```